src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -221,6 +222,29 @@ namespace ngraph
                 writer << "ctx->mlsl_env->Wait(req);\n";
                 writer.block_end();
             }
+
+            template <>
+            void CPU_Emitter::EMITTER_DECL(ngraph::op::DistBroadcast)
+            {
+                const element::Type& element_type = args[0].get_element_type();
+                auto data_type = "MLSL::DT_FLOAT";
+
+                if (element_type == element::f32)
+                {
+                    data_type = "MLSL::DT_FLOAT";
+                }
+                else if (element_type == element::f64)
+                {
+                    data_type = "MLSL::DT_DOUBLE";
+                }
+
+                writer.block_begin();
+                writer << "MLSL::CommReq* req = ctx->mlsl_dist->Bcast(" << args[0].get_name()
+                       << ", " << args[0].get_size() << ", " << data_type << ", " << 0
```

Comment:
Don't you need quotes for 0?

Reply:
The quotes for 0 are not necessary but the format is much better with 0 quoted.